lib/src/main/java/com/auth0/android/lock/views/ValidatedInputView.java

```diff
@@ -225,7 +225,7 @@ private void setupInputValidation() {
                 input.setInputType(InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS);
                 inputIcon = R.drawable.com_auth0_lock_ic_username;
                 hint = getResources().getString(R.string.com_auth0_lock_hint_username);
-                error = getResources().getString(R.string.com_auth0_lock_input_error_empty);
+                error = getResources().getString(R.string.com_auth0_lock_input_error_username_empty);
```

Comment:
Is this a breaking change since we removed an error string? Also if we are doc them we should update that doc